Apps/Src/UpdateVelocity.c

```diff
@@ -0,0 +1,36 @@
+#include "SendTritium.h"
+#include "CarState.h"
+#include "UpdateVelocity.h"
+
+// As of 03/13/2021 the function just returns a 1 to 1 conversion
+static convertPedaltoMotorPercent(uint8_t pedalPercentage){
```

Comment:
This function needs to return a float in the range of 0.0 to 1.0, so you probably want to divide the pedal percentage by 100.0.